### DISCUSSION TITLE
Where is the VS2022 Branch?

### DISCUSSION THREAD

**Discussion by thewolfprod** [CosmosOS/Cosmos#2362] [Q&A]
I typed `git clone https://github.com/CosmosOS/Cosmos.git --branch VS2022` into PowerShell but its saying that `Remote branch VS2022 not found in upstream origin`

**Comment by xyve7** [comment]
It's part of the main/master branch iirc, just clone it normally.

**Reply by thewolfprod** [reply]
Thanks, but i have another problem and I can't find solution. Im getting error `Visual Studio Workloads was not found. Install Visual Studio Extension Development` - and I don't really know what it is and where find this... Google doesn't say anything.

**Reply by thewolfprod** [reply]
Ahh okey, for future:
U need to install this
![This](https://user-images.githubusercontent.com/55655314/181783630-5a48e922-bbfa-40bd-81c0-3db86f9c9799.png)
